mmtrack/models/sot/mixformer.py

```diff
@@ -0,0 +1,398 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+from addict import Dict
+from mmdet.core.bbox.transforms import bbox_xyxy_to_cxcywh
+from mmdet.models.builder import build_backbone, build_head
+from torchvision.transforms.functional import normalize
+
+from ..builder import MODELS
+from .base import BaseSingleObjectTracker
+
+
+@MODELS.register_module()
+class MixFormer(BaseSingleObjectTracker):
```

Comment:
I noticed that this module is similar to `STARK`. Can it inherent `STARK` module?